kit/Kit.cpp

```diff
@@ -3126,6 +3163,33 @@ void runKitLoopInAThread()
 
 #endif // !BUILDING_TESTS
 
+void consistencyCheckJail()
+{
+    static size_t warned = 0;
+    if (!(warned++))
```

Comment:
Don't we want to set this to true (or ++) only when we do actually detect inconsistency (and report) ?

Meaning, we do want to check as long as we don't see any issues, but then disable the check from that point on.